09-spring-boot-ehcache-xml/src/main/resources/ehcache.xml

```diff
@@ -0,0 +1,63 @@
+<!--
+  Copyright (c) 2023 Software AG, Darmstadt, Germany and/or Software AG USA Inc., Reston, VA, USA, and/or its subsidiaries and/or its affiliates and/or their licensors.
+  Use, reproduction, transfer, publication or disclosure is prohibited except as specifically provided for in your License Agreement with Software AG.
+-->
+<config
+    xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance'
+    xmlns='http://www.ehcache.org/v3'
+    xmlns:jsr107='http://www.ehcache.org/v3/jsr107'
+    xsi:schemaLocation="
+        http://www.ehcache.org/v3 http://www.ehcache.org/schema/ehcache-core-3.0.xsd
+        http://www.ehcache.org/v3/jsr107 http://www.ehcache.org/schema/ehcache-107-ext-3.0.xsd">
+
+  <service>
+    <jsr107:defaults enable-statistics="true"/>
+  </service>
+
+  <cache alias="rates">
+    <key-type>java.lang.String</key-type>
+    <value-type>java.lang.Double</value-type>
+    <expiry>
+      <ttl unit="seconds">30</ttl>
+    </expiry>
+    <listeners>
+      <listener>
+        <class>com.example.springbootehcache.caching.CacheEventRedirect</class>
+        <event-firing-mode>ASYNCHRONOUS</event-firing-mode>
+        <event-ordering-mode>ORDERED</event-ordering-mode>
+        <events-to-fire-on>CREATED</events-to-fire-on>
+        <events-to-fire-on>UPDATED</events-to-fire-on>
+        <events-to-fire-on>EXPIRED</events-to-fire-on>
+        <events-to-fire-on>REMOVED</events-to-fire-on>
+        <events-to-fire-on>EVICTED</events-to-fire-on>
+      </listener>
+    </listeners>
+    <resources>
+      <offheap unit="MB">1</offheap>
+    </resources>
+  </cache>
+
+  <cache alias="currencies">
+    <key-type>java.lang.String</key-type>
+    <value-type serializer="com.example.springbootehcache.caching.CurrenciesSerializer">java.util.ArrayList</value-type>
```

Comment:
The serialiser is a spring beans containing some `@Autowire` and depending on Spring injection